tkltest/util/unit/config_options_unit.py

```diff
@@ -207,15 +208,16 @@ def __conditionally_required(opt_name, config):
                             '(default: app classes are instrumented online)'
         },
         'build_type': {
-            'required': False,
+            'required': __conditionally_required,
             'is_toml_option': True,
             'is_cli_option': True,
             'short_name': '-bt',
             'long_name': '--build-type',
             'type': str,
-            'choices': ['ant', 'maven',  'gradle'],
-            'default_value': 'ant',
-            'help_message': 'build file type for compiling and running the tests: ant, maven, or gradle'
+            'choices': ['ant', 'maven',  'gradle', ''],
+            'default_value': '',
```

Comment:
why is the default value empty? I thought we agreed the default value will be maven. And it should not be required because it has a default value.

Reply:
As discussed - I will change it to not required, with default value `maven`, and empty string won't be one of its possible values.